src/sdk/conference/channel.js

```diff
@@ -61,6 +59,7 @@ export class ConferencePeerConnectionChannel extends EventDispatcher {
     this._sdpResolverMap = new Map(); // internalId => {finish, resolve, reject}
     this._sdpResolvers = []; // [{finish, resolve, reject}]
     this._sdpResolveNum = 0;
+    this._remoteMediaStream = new Map(); // Key is subscription ID, value is MediaStream.
```

Comment:
this._remoteMediaStreams  instead of this._remoteMediaStream

Reply:
Fixed.